sql/V1_10__credit_classes.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE IF NOT EXISTS
+  credit_classes (
```

Comment:
This table is not storing credit classes, only class issuers. We should make this more clear.

```suggestion
  class_issuers (
```